### PR TITLE
Fix test and proper error handler

### PR DESCRIPTION
**Pull request by sublet**
Test was failing because of the caseId change.  All better now.

Also took the opportunity to add an error handler.

### CODE REVIEW

File: test/integration/case.test.js [hunk 1]
```diff
@@ -51,12 +51,12 @@ describe('Case', () => {
       jwtSecret.secret,
     );
 
-    const caseParams = {
-      organization_id: currentOrg.id,
-      state: 'unpublished'
-    };
+    // const caseParams = {
```
Comment (MarshallMcCoy) [inline test/integration/case.test.js:54]: Should this still be commented out? If so, do we event need it?

Comment (sublet) [reply test/integration/case.test.js]: Blargh...handling.

**Comment by sublet** [comment]
@AdamLeonSmith there is an error on `_destroySSL` in one of the tests.  Not sure what that is but will let you handle.

**Comment by AdamLeonSmith** [comment]
The test failure doesn't look related to this PR.  So don't let it block the merge.

@sherifkozman ping me on this when you have a minute

**Comment by sublet** [comment]
Thanks @AdamLeonSmith ... will do.

**Pull request merged**